aldryn_events/tests/test_docs.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import with_statement
+from contextlib import contextmanager
+
+import os
+import socket
+import sys
+
+import aldryn_events
+
+from django.utils.six.moves import StringIO
+from sphinx.application import Sphinx
+
+try:
+    import enchant
+except ImportError:
+    enchant = None
+
+from cms.test_utils.compat import skipIf
+from cms.test_utils.testcases import CMSTestCase
+from cms.test_utils.util.context_managers import TemporaryDirectory
+
+
+ROOT_DIR = os.path.dirname(aldryn_events.__file__)
+DOCS_DIR = os.path.abspath(os.path.join(ROOT_DIR, u'..', u'docs'))
+
+
+def has_no_internet():
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        s.settimeout(5)
+        s.connect(('4.4.4.2', 80))
+        s.send(b"hello")
```

Comment:
It would be best not to mix quoting like this. The general rule is to always use `'`, unless you need to quote a `'` - if you need to do that you start with `"` so that you don't need to backslash-quote.